feature/interface/aggregate/otg_tests/aggregate_all_not_viable_test/README.md

```diff
@@ -0,0 +1,116 @@
+                                                 '
+                                                LAG_3
+
+```
+
+- Connect ATE/ATE port-1 to DUT port-1, and ATE/ATE ports 2 through 7 to DUT ports 2-7,
+  and ATE/ATE ports 8, 9 to DUT ports 8, 9
+- Configure ATE/ATE and DUT ports 1 to be LAG_1 w/ LACP running.
+- Configure ATE/ATE and DUT ports 2-7 to be LAG_2 w/ LACP running.
+- Configure ATE/ATE and DUT ports 8-9 to be LAG_3 w/ LACP running.
+- Establish ISIS adjacencies on LAG_1, LAG_2, LAG_3.
+  1. Advertise one network prefix (pfx1) from ATE/ATE LAG_1
+  1. Advertise one network prefix (pfx2) from ATE/ATE LAG_2 and ATE/ATE LAG_3.
+- Establish iBGP between ATE and DUT over LGA_1 using LAG_1 interface IPs and advertise prefix pfx3 with BGP NH from pfx2 range.
+- Programm via gRIBI route for prefix pfx4 with single NHG pointing LAG_2 (al
+  ports are forwarding-viable at this point).
+- For both (ISIS cost of LAG_2 equal to ISIS cost of LAG_3) AND (ISIS cost of
+  LAG_2 lower than ISIS cost of LAG_3):
+  - Run traffic:
+    - From prefix pfx1 to all three: pfx2, pfx3, pfx4
+    - From prefix pfx2 to: pfx1
+  - Make the forwarding-viable transitions from TRUE --> FALSE on ports 3-7
+    within the LAG_2 on the DUT
+    - ensure that only DUT port 2 of LAG ports has bidirectional traffic.
+    - Ensure there is no traffic transmitted out of DUT ports 3-7
+    - ensure that traffic is received on all port2-7 and delivered to ATE/ATE port1
+    - ensure there are no packet losses in steady state (no congestion).
+    - Ensure there is no traffic received on DUT LAG_3
+    - Ensure there is no traffic transmitted on DUT LAG_3
+  - Disable/deactive laser on ATE port2; All LAG_2 members are either down (port2) or
+    set with forwarding-viable=FALSE
+    - Ensure ISIS adjacency is UP on DUT LAG_2 and ATE LAG_2
+    - Ensure there is no traffic transmitted out of  DUT ports 2-7 (LAG_2)
+    - ensure that traffic is received on all port3-7 and delivered to ATE/ATE LAG_1
+    - ensure there are no packet losses in steady state (no congestion) for
+      traffic prom ATE LAG_2 to ATE LAG_1 (pfx_1).
+    - ensure there are no packet losses in steady state (no congestion) for
+      traffic from ATE LAG_1 to ATE LAG_3 (pfx_2, pfx3).
+    - Ensure there is no traffic received on DUT LAG_3
+    - Ensure that traffic from ATE port1 to pfx2, pfx3 are transmitted via DUT
+      LAG3
+    - Ensure that traffic from ATE port1 to pfx4 are discarded on DUT
+  - Make the forwarding-viable transitions from FALSE --> TRUE on a ports 7
+    within the LAG_2 on the DUT
+    - ensure that only DUT port 7 of LAG ports has bidirectional traffic.
+    - Ensure there is no traffic transmitted out of  DUT ports 2-6
+    - ensure that traffic is received on all port3-7 and delivered to ATE/ATE port1
+    - ensure there are no packet losses in steady state (no congestion).
+    - Ensure there is no traffic received on DUT LAG_3
```

Comment:
same, why no traffic on `LAG_3`?

Reply:
I make separate test procedure for cahe when cost of LAG_2 < LAG_3 and separate when costs are equall.